plugins/out_azure_blob/azure_blob.c

```diff
@@ -979,6 +997,8 @@ static void cb_azure_blob_flush(struct flb_event_chunk *event_chunk,
      */
     ret = ensure_container(ctx);
     if (ret == FLB_FALSE) {
+        flb_plg_error(ctx->ins, "cannot ensure container '%s' exists",
```

Comment:
This line could generate confusion when `ensure_container` exits through line 600 and it seems that those code paths already report the result so it's not necessary to do it here.

Reply:
Yes, I agree it's redundant and can cause confusion. I removed it and changed the log message in [line 603 ](https://github.com/fluent/fluent-bit/pull/9457/files#diff-f844d79a27a0ee277d33bbdd76e0244cf5c89af81e86a1ecdf2577f11f3cef1fR603) to be error instead of debug.